eng/versioning/version_client.txt

```diff
@@ -140,11 +140,12 @@ com.microsoft.azure:spring-cloud-azure-appconfiguration-config;1.3.0;1.4.0-beta.
 com.microsoft.azure:spring-cloud-azure-feature-management-web;1.3.0;1.4.0-beta.1
 com.microsoft.azure:spring-cloud-azure-feature-management;1.3.0;1.4.0-beta.1
 com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config;1.3.0;1.4.0-beta.1
-com.azure.spring:azure-spring-cloud-appconfiguration-config-web;2.0.0;2.1.0
-com.azure.spring:azure-spring-cloud-appconfiguration-config;2.0.0;2.1.0
+com.azure.spring:azure-spring-cloud-appconfiguration-config-web;2.1.0;2.2.0-beta.1
+com.azure.spring:azure-spring-cloud-appconfiguration-config;2.1.0;2.2.0-beta.1
 com.azure.spring:azure-spring-cloud-feature-management-web;2.0.0;2.1.0-beta.1
 com.azure.spring:azure-spring-cloud-feature-management;2.0.0;2.1.0-beta.1
-com.azure.spring:azure-spring-cloud-starter-appconfiguration-config;2.0.0;2.1.0
+com.azure.spring:azure-spring-cloud-starter-appconfiguration-config;2.1.0;2.2.0-beta.1
+com.azure.spring:azure-identity-spring;1.8.0;1.9.0-beta.1
```

Comment:
don't need this `com.azure.spring:azure-identity-spring;1.8.0;1.9.0-beta.1` anymore.